scripts/exgdas_atmos_verfrad.sh

```diff
@@ -0,0 +1,225 @@
+#! /usr/bin/env bash
+
+source "$HOMEgfs/ush/preamble.sh"
+
+################################################################################
+####  UNIX Script Documentation Block
+#                      .                                             .
+# Script name:         exgdas_vrfyrad.sh
+# Script description:  Runs data extract/validation for global radiance diag data
+#
+# Author:        Ed Safford       Org: NP23         Date: 2012-01-18
+#
+# Abstract: This script runs the data extract/validation portion of the 
+#           RadMon package.  
+#
+# Condition codes
+#       0 - no problem encountered
+#      >0 - some problem encountered
+#
+################################################################################
+
+export VERBOSE=${VERBOSE:-YES}
+
+export RUN_ENVIR=${RUN_ENVIR:-nco}
+export NET=${NET:-gfs}
+export RUN=${RUN:-gdas}
+export envir=${envir:-prod}
+export COMPONENT=${COMPONENT:-atmos}
+
+#  Command line arguments
+export PDY=${1:-${PDY:?}} 
+export cyc=${2:-${cyc:?}}
+
+#  Directories
+export COM_IN=${COM_IN:-$(compath.py ${envir}/${NET}/${gfs_ver})}
+export COMIN=${COMIN:-$COM_IN/${RUN}.${PDY}/${cyc}/$COMPONENT}
+
+
+#  Filenames
+export biascr=${biascr:-$COMIN/gdas.t${cyc}z.abias}
+export radstat=${radstat:-$COMIN/gdas.t${cyc}z.radstat}
+export satype_file=${satype_file:-${FIXgdas}/gdas_radmon_satype.txt}
+
+#  Other variables
+export RAD_AREA=${RAD_AREA:-glb}
+export MAKE_CTL=${MAKE_CTL:-1}
+export MAKE_DATA=${MAKE_DATA:-1}
+export USE_ANL=${USE_ANL:-1}
+export PDATE=${PDY}${cyc}
+export DO_DIAG_RPT=${DO_DIAG_RPT:-1}
+export DO_DATA_RPT=${DO_DATA_RPT:-1}
+export USE_MAIL=${USE_MAIL:-0}
```

Comment:
The capability to mail warning reports directly from the data extraction was removed a while ago so the USE_MAIL, MAIL_TO, and MAIL_CC variables can be removed.

Reply:
Removed